content/en/docs/ambient/architecture/index.md

```diff
@@ -9,15 +9,66 @@ owner: istio/wg-networking-maintainers
 test: n/a
 ---
 
+## Ambient APIs
+
+### Labels
+
+You can use the following labels to enroll your namespace to ambient, enroll your resource to use a waypoint and specify, the capture scope of your waypoint.
```

Comment:
Suggestion for consistent terms.....

"add your [namespace | service | workload | pod ...] to the mesh"
"enable a waypoint for traffic to your [namespace | service | workload | pod ...] "
"control what traffic is sent to the waypoint"

I'd avoid the use of the term 'enroll' as it's harder to conjugate with 'to' making statements harder to form.

Reply:
cc @MorrisLaw I had enroll earlier because we have the `istioctl x waypoint --enroll-namespace` cmd. thoughts?  `enable` does sound easier to understand.

Reply:
updated

Reply:
We can call the `--enroll-namespace` command to enable a waypoint for traffic to your namespace or use the `use-waypoint` annotation to enable a waypoint for traffic to a [service | workload | pod ...]. That kind of wording makes sense to me here for readability and better understanding. I like it, good call!